posthog/helpers/session_recording.py

```diff
@@ -250,6 +258,60 @@ def get_active_segments_from_event_list(
     return active_recording_segments
 
 
+# List of properties from the event payload we care about for our uncompressed `events_summary`
+# NOTE: We should keep this as minimal as possible
+EVENT_SUMMARY_DATA_INCLUSIONS = [
```

Comment:
Weird positioning of a const - indicates it might be worthwhile splitting this file?

Reply:
Yeah moved it around. Will definitely be moving all this around later